### DISCUSSION TITLE
checkboxGroupInput in R Shiny

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#623] [Q&A] [answered]
Saya punya kodingan seperti ini di R Studio Shiny : 

Bagian UI
<img width="520" alt="image" src="https://user-images.githubusercontent.com/106010284/231246017-1c89793c-92a5-4f28-bc47-26e9c7766796.png">

Bagian Server
<img width="706" alt="image" src="https://user-images.githubusercontent.com/106010284/231246209-3d3b33df-6d13-4af2-af17-ff1d811995c9.png">

Saya ingin ketika checkboxnya dipilih Attack Type, maka plot hanya menampilkan attack_custom saja. Ketika checkboxnya dipilih Attack Target, maka plot hanya menampilkan policy_name_custom saja. Akan tetapik, ketika checkboxnya dipilih Attack Type dan Attack Target, maka plotnya menampilkan attack_target dan policy_name.

Mohon bantuannya kak, terima kasih 🙏

**Comment by ramdhinafinita** [comment, marked as answer]
Dear Pak @yohanespm77 
Dari code yang Anda lampirkan tidak ada kesalahan pada koneksi antara inputId yang dijadikan checkboxnya dan versi render nya di server. Apakah Anda bisa melampirkan code full berupa zip agar kami bisa tracking errornya?

Selain itu, apakah Anda bisa menampilkan tampilan hasil yang telah Anda coba buat?

Terima kasih.

**Reply by yohanespm77** [reply]
Maaf kak, sudah bisa. Saya menggunakan if condition. Terima kasih atas repsonnya kak.
